WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m

```diff
@@ -155,6 +156,20 @@ - (CGRect)wdNativeFrame
   return self.frame;
 }
 
+/**
+ Returns a comma-separated string of accessibility traits for the element.
+ This method converts the element's accessibility traits bitmask into human-readable strings
+ using fb_accessibilityTraitsToStringsArray. The traits represent various accessibility
+ characteristics of the element such as Button, Link, Image, etc.
+ 
+ @return A comma-separated string of accessibility traits, or an empty string if no traits are set
```

Comment:
maybe add a link to https://developer.apple.com/documentation/uikit/uiaccessibilitytraits?language=objc for better overview